playwright.config.ts

```diff
@@ -0,0 +1,47 @@
+import { PlaywrightTestConfig } from '@playwright/test'
+import { devices } from '@playwright/test'
+import dotenv from 'dotenv'
+import * as path from "path"
+import * as envalid from "envalid"
+
+dotenv.config()
+const env = envalid.cleanEnv(process.env, {
+  WEBPACK_PORT: envalid.port()
+})
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  testDir: './tests-e2e',
+  /* Maximum time one test can run for. */
+  timeout: 30 * 1000,
+  expect:  {
+    timeout: 5000
+  },
+  fullyParallel: true,
+  forbidOnly:    !!process.env.CI,
+  retries:       process.env.CI ? 2 : 0,
+  workers:       process.env.CI ? 1 : undefined,
+  reporter:      'html',
+  use:           {
+    actionTimeout: 0,
+    trace:         'on-first-retry',
+  },
```

Comment:
BTW: Most of these settings are what Playwright recommends. I created this config file via (https://playwright.dev/docs/intro#installing-playwright)
```
yarn create playwright
```
and these were the settings it gave me. My customizations were:
- removing some comments because it seemed cluttery and TS docstrings / playwright docs have the same information.
- removing the extra browsers (firefox, webkit)
- customizing the `webServer` portion. 

We could restore firefox + webkit. I'm indifferent. I'm not sure how useful the extra browsers would be at the moment; maybe if we were doing visual or very interactive tests.